internal/cli/plugin/update.go

```diff
@@ -0,0 +1,265 @@
+// Copyright 2024 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package plugin
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path"
+
+	"github.com/google/go-github/v61/github"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/cli/require"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/flag"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/log"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/plugin"
+	"github.com/mongodb/mongodb-atlas-cli/atlascli/internal/usage"
+	"github.com/spf13/cobra"
+)
+
+var (
+	errTooManyArguments        = errors.New(`either the "--all" flag or the plugin identifier can be provided, but not both`)
+	errNotEnoughArguments      = errors.New(`either the "--all" flag or the plugin identifier needs to be provided`)
+	errPluginHasNoGithubValues = errors.New(`specified plugin does not have any Github values`)
```

Comment:
This error message is not super clear. Could you add something like "are you sure you installed this plugin using `plugin install`?". (Just a suggestion, I'm sure there's a better way to phrase this)

Reply:
done